README.md

```diff
@@ -5,7 +5,126 @@
 
 ## Getting started
 
-TODO
+|Package| Download Link                                                                                        | Description                                                                |
+|----|------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------|
+|Workleap.DomainEventPropagation.Abstractions| ![nuget](https://img.shields.io/nuget/v/Workleap.DomainEventPropagation.Abscratcions.svg?logo=nuget) | Contains abstractions that are used for publishing and recieveing messages |
+|Workleap.DomainEventPropagation.Publishing| ![nuget](https://img.shields.io/nuget/v/Workleap.DomainEventPropagation.Publishing.svg?logo=nuget)   | Contains classes for message publishing                                    |
+|Workleap.DomainEventPropagation.Subscription| ![nuget](https://img.shields.io/nuget/v/Workleap.DomainEventPropagation.Subscription.svg?logo=nuget) |  Contains classes for subscriptions                                        |
+
+### What does the Workleap.DomainEventPropagation package do?
+* Provides list of existing Topics
+* Serializes domain events
+* Manages publication fields
```

Comment:
I'm not sure to understand what this means although I have some knowledge of the lib and Event Grid, could we add details and try to read this as someone that doesn't really know anything?